crates/ef-testing/src/utils/assert.rs

```diff
@@ -56,7 +57,8 @@ pub fn assert_contract_post_storage(
                 .copied()
                 .unwrap_or_default();
 
-            if actual_state_value != StarkFelt::from(value) {
+            let value = StarkFelt::from(value);
```

Comment:
does this change something?

Reply:
Just allows to have the same type and output both values as hex when the comparison fails